core/modules/file/file.entity.inc

```diff
@@ -126,6 +128,58 @@ class File extends Entity {
       'options' => array(),
     );
   }
+
+  /**
+   * Overrides Entity::createAccess().
+   */
+  public static function createAccess($bundle = NULL, $account = NULL) {
+    // If no user object is supplied, the access check is for the current user.
+    if (empty($account)) {
+      $account = $GLOBALS['user'];
+    }
+
+    return user_access('access content', $account);
+  }
+
+  /**
+   * Overrides Entity::access().
+   *
```

Comment:
It would be good to note that "view" is not supported on the File Entity itself because it has no page callback (yet), but there is separately `file_download_access()` to check if a file is downloadable.